core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -126,6 +132,10 @@ oppia.directive('feedbackPopup', [
               getTriggerElt().trigger('click');
             });
           };
+
+          $scope.$on('$destroy', function() {
+            BackgroundMaskService.activateMask(false);
```

Comment:
This reads a bit weirdly. Maybe have two methods: activateMask and deactivateMask.